tests/estimators/online/general/test_ukf.py

```diff
@@ -0,0 +1,212 @@
+import numpy as np
+from pytest import fixture
+
+from asoh.estimators.online import ModelFilterInterface, ControlVariables, OutputMeasurements
+from asoh.estimators.online.general.kalman.unscented import UnscentedKalmanFilter as UKF
+from asoh.estimators.online.general.kalman import KalmanHiddenState
+
+
+# Define Lorenz dynamics
+class Lorenz(ModelFilterInterface):
```

Comment:
A thought about this test case: do you think we can still use the Lorenz model if we start making the `ModelFilterInterface` more closely integrated with `CellModel`? Can we make a `CellModel` that implements the Lorenz equations?

Reply:
I thought a bit about that, but I think it would be weird to have a `CellModel`, with `abstractmethod`s `update_transient_state` and `calculate_terminal_voltage`, that does not do anything related to a battery cell.

Reply:
Let's see where we need to change it when we get there. I really would like to prevent users from needing to implement the same loops for `ModelFilterInterface` subclasses continually.